integration-tests/environments/react-native/ios/Podfile

```diff
@@ -30,10 +30,16 @@ target 'RealmReactNativeTests' do
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable the next line.
   # use_flipper!()
-  # FIXME: Applying https://github.com/facebook/flipper/issues/834#issuecomment-830833919
-  use_flipper!({ 'Flipper-Folly' => '2.5.3', 'Flipper' => '0.87.0', 'Flipper-RSocket' => '1.3.1' })
```

Comment:
Yes, I can activate it.  I was just cutting the build time in half 😆 I

Reply:
Yeah, that's a bummer 🤔 Perhaps we can detect a CI environment variable and do it conditionally based on that?

Reply:
Good idea.  I'll add that

Reply:
Why not use the `CI` variable which is already set on GitHub Actions? This would have the added benefit that our developers won't have to set a variable to enable Flipper, but the other way around.

Also, are you sure you can ditch the version constraints?

Reply:
Those version constraints are a workaround for a bug in React Native relating to an update in xcode.  They are no longer necessary

Reply:
Would it be more preferable to have it enabled by default?

Reply:
So now the logic is as follows:
* Flipper is disabled for Github Actions (CI=true)
* Flipper can be disabled with the variable DISABLE_FLIPPER (DISABLE_FLIPPER=true)
* `pod install` will add flipper support if none of these variables are true.